### PR TITLE
Option to disable logging of stopwatches in audit log.

### PR DESCRIPTION
**Pull request by marcstern**
[Issue #1067 - Marc Stern]

### CODE REVIEW

**Comment by zimmerle** [comment]
Please make sure that the JSON audit log is also covered.

**Comment by marcstern** [comment]
done

File: CHANGES [hunk 1]
```diff
@@ -1,7 +1,7 @@
 DD MMM YYYY - 2.9.2 - To be released
 ------------------------------------
 
- * {dis|en}able-filename-logging: Option to disable logging of stopwatches
```
Comment (zimmerle) [inline CHANGES]: Same as others: make sure to not perform a fix on the top of your own fix. Try to keep this feature a one-patch thing.

File: apache2/msc_logging.c [hunk 2]
```diff
@@ -1,2285 +1,2288 @@
-/*
```
Comment (zimmerle) [inline apache2/msc_logging.c]: Ops :)

**Comment by zimmerle** [comment]
Closing as explained in the comments.

File: apache2/msc_logging.c (hunk 2 above)
Comment (marcstern) [inline apache2/msc_logging.c]: This seems the \r\n problem.
I explicitely replaced all \r\n by \r but git can't see any diff to commit.
What should I do?

Comment (marcstern) [inline apache2/msc_logging.c]: I found the way to fix it!

**Pull request closed without merging**